fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/Repository.java

```diff
@@ -213,6 +245,27 @@ public Model getObjectModel(URI metadataUri) throws FedoraException {
 		}
 	}
 
+	/**
+	 * 
+	 * @param subject
+	 * @param property
+	 * @param object
+	 */
+	public void createRelationship(PID subject, Property property, Resource object) {
+		String sparqlUpdate = RDFModelUtil.createSparqlInsert(subject.getRepositoryPath(), property, object);
```

Comment:
They return more or less the same thing, getRepositoryPath returns the absolute path to object, while getRepositoryURI returns the same thing but as a URI object.  The reason for having both is just that there are two apis involved, one that demands strings and the other demands URIs